HousingManagementSystemApi/Helpers/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+namespace HousingManagementSystemApi.Helpers;
+
+using System.Collections.Generic;
+using System.Linq;
+// using Ardalis.GuardClauses;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.DependencyInjection;
+
+public static class ServiceCollectionExtensions
+{
+    public static void AddCosmosAddressContainers(this IServiceCollection services, IEnumerable<string> repairType)
+    {
+        // Guard.Against.NullOrWhiteSpace(repairType, nameof(repairType));
```

Comment:
You can still use Guard.Against using 

foreach (var repairType in repairTypes)
        {
            Guard.Against.NullOrWhiteSpace(repairType, nameof(repairType));
            Guard.Against.InvalidInput(repairType, nameof(repairType), RepairType.IsValidValue);
        }